jkube-kit/doc/src/main/asciidoc/inc/_enricher.adoc

```diff
@@ -44,12 +47,14 @@ This plugin comes with a set of default enrichers. In addition custom enrichers
 | Enricher | Description
 
 | <<jkube-configmap-file>>
-| Add ConfigMap elements defined as XML or as annotation.
+| Add ConfigMap elements defined as Groovy or as annotation.
+
+ifeval::["{plugin-type}" == "maven"]
```

Comment:
I understand this is added because we still don't support the rest of enrichers, right?
Maybe add a TODO comment, so that we can keep moving the if clause down until we finally remove it.